### PR TITLE
add MAP URLs to local cypress non-smoke tests

### PR DESCRIPTION
**Pull request by pjlee11**
Part of #3455

**Overall change:** Cypress tests config for services with MAP assets for local non-smoke tests.

**Code changes:**

- Adds a URL for each service (minus /news, /naidheachdan, /cymrufyw and /sport) to enable non-smoke Cypress tests on local environment
  - Note: Ukchina, Zhongwen and Serbian will be added in https://github.com/bbc/simorgh/pull/3775
- Changes `Cypress.env('APP_ENV') === 'local'` in mediaAssetPage cypress config to `Cypress.env('APP_ENV') !== 'test' || Cypress.env('APP_ENV') !== 'live'` for consistency 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- ~[ ] This PR requires manual testing~

### CODE REVIEW

File: cypress/support/config/services.js [hunk 1]
```diff
@@ -27,9 +27,15 @@ module.exports = {
           Cypress.env('APP_ENV') === 'live'
             ? undefined
             : '/afaanoromoo/bbc_afaanoromoo_radio/liveradio',
-        smoke: true,
+        smoke: false,
```
Comment (pjlee11) [inline cypress/support/config/services.js:30]: Not sure why this was set to true but for consistency setting it to false

Comment (jroebu14) [reply cypress/support/config/services.js:30]: Is it not something to do with having one nominated service for that page type to be smoke tested in CI?

**Review by ryanmccombe — APPROVED** [review]

**Comment by ryanmccombe** [comment]
> Resolves #NUMBER

File: cypress/support/config/services.js (hunk 1 above)
Comment (pjlee11) [reply cypress/support/config/services.js:30]: Potentially, I don't know the requirements behind it. My understanding is that we are only smoke testing pages that are in production. Happy either way, just seemed inconsistent with the rest of the config

Comment (jroebu14) [reply cypress/support/config/services.js:30]: Yeh I think it is true for the reason I mentioned. @benjaminhobbs can confirm

**Review by dr3 — APPROVED** [review]
Not run them or checked the URLs, but LGTM. Please vouch they pass on all envs before merging

**Review by ibMadbouly — APPROVED** [review]

**Pull request merged**